### PR TITLE
PCI-1067 Enable Dependabot Docker to auto-merge the PR

### PR DESCRIPTION
**Pull request by aliculPix4D**
- enables Dependabot Docker to auto-merge the PR it opens
- delete the feature branch it created

TO DO BEFORE MERGING: 
- [x] remove equivalent job from `set-pipeline job` in `github-automation-tools`

### CODE REVIEW

**Comment by Pix4D-Janus** [comment]
Your pipeline is set.
Check it here:https://builder.ci.pix4d.com/teams/developers/pipelines/dependabot-PCI-1067-auto-merge-dependabot

File: docker/dependabot-main.rb [hunk 1]
```diff
@@ -41,6 +41,24 @@ def requirements(checker)
   requirements
 end
 
+# Automatically merge a pull request for docker feature package
```
Comment (marco-m-pix4d) [inline docker/dependabot-main.rb]: What is the meaning of "docker feature package" ? Can we find a better description ?

Comment (aliculPix4D) [reply docker/dependabot-main.rb]: At the top of the file we have:
```
# LIST OF ENVIROMENTAL VARIABLES NEEDED:
# FEATURE_PACKAGE i.e docker or concourse
```
so `feature package` can be set to either `docker` or `concourse`. 
For sure, I will add a better description here. Just for info: you have already told me to note that we need a better name instead `FEATURE_PACKAGE`

**Comment by aliculPix4D** [comment]
Example PR opened by Dependabot and auto-merged, after which feature branch is deleted:
https://github.com/Pix4D/test-dependabot-docker/pull/84

File: docker/dependabot-main.rb (hunk 1 above)
Comment (aliculPix4D) [reply docker/dependabot-main.rb]: Done as suggested.

**Pull request merged**